### PR TITLE
feat: add dox for dtkdevice(dtklsdevice)

### PR DESCRIPTION
**Pull request by wyu71**
add dox for dtkdevice(dtklsdevice)

Log: add dox for dtkdevice(dtklsdevice)

### CODE REVIEW

**Comment by toberyan** [comment]
+1

File: dtklsdevice/docs/dmemorydevice.zh_CN.dox [hunk 1]
```diff
@@ -0,0 +1,104 @@
+// SPDX-FileCopyrightText: 2022 UnionTech Software Technology Co., Ltd.
+//
+// SPDX-License-Identifier: LGPL-3.0-or-later
```
Comment (xzl01) [inline dtklsdevice/docs/dmemorydevice.zh_CN.dox:3]: dox文件不需要加这个头 只需要在dep5 文件中注明即可 并且为cc by 4.0协议

Comment (wyu71) [reply dtklsdevice/docs/dmemorydevice.zh_CN.dox]: 已安建议修改

**Review by toberyan — APPROVED** [review]

**Pull request merged**